packages/clay-navigation/src/index.tsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import {Breadcrumb} from './Breadcrumb';
+
+const ClayNavigation: React.FunctionComponent<
+	React.HTMLAttributes<HTMLDivElement>
+> & {
+	Breadcrumb: typeof Breadcrumb;
+} = ({className, ...otherProps}) => {
+	return (
+		<div {...otherProps} className={classNames(className)}>
```

Comment:
Is this just essentially a placeholder for now?

Reply:
Nope, I miss it.